README.md

```diff
@@ -38,7 +38,7 @@ Apache Doris is an easy-to-use, high-performance and real-time analytical databa
 Based on this, Apache Doris can better meet the scenarios of report analysis, ad-hoc query, unified data warehouse, Data Lake Query Acceleration, etc. Users can build user behavior analysis, AB test platform, log retrieval analysis, user portrait analysis, order analysis, and other applications on top of this.
 
 
-🎉 Version 1.1.2 released now! Check out the 🔗[Release Notes](https://doris.apache.org/docs/releasenotes/release-1.1.2) here. 
+🎉 Version 9999.99999 released now! Check out the 🔗[Release Notes](https://doris.apache.org/docs/releasenotes/release-1.1.2) here. 
```

Comment:
Please do not arbitrarily modify the correct document, this will cause your PR will not be merged